testkit/tests/system_api.rs

```diff
@@ -19,20 +19,21 @@ extern crate pretty_assertions;
 
 use exonum::{
     api::node::{
-        private::NodeInfo, public::system::{ConsensusStatusInfo, HealthCheckInfo},
+        private::NodeInfo, public::system::{ConsensusStatus, ConnectivityStatus, HealthCheckInfo},
     },
     helpers::user_agent, messages::PROTOCOL_MAJOR_VERSION,
 };
 use exonum_testkit::{ApiKind, TestKitBuilder};
 
 #[test]
-fn test_healthcheck_connectivity_false() {
+fn test_healthcheck_status_false() {
```

Comment:
And one more: `test_helthcheck_status_false` name doesn't correlate with test contents.